crates/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -315,6 +344,32 @@ fn copy_dir_recursive(
     Ok(())
 }
 
+/// Use pkgdb to resolve a manifest into a lockfile
+pub fn resolve_manifest_to_lockfile(
```

Comment:
nit (non-blocking): are we really speaking of "resolving" a manifest into a lockfile or should this rather be "lock_manifest"? Technically the elements in the manifest are already "resolved" just not locked/pinned to a specific version/revision...

Reply:
You're right, I think "locking" makes more sense